.github/workflows/_linux-test-stable-fa3.yml

```diff
@@ -0,0 +1,256 @@
+# The point of this workflow is to test that a FA3 wheel that was built based off the
```

Comment:
Why not do it inside `test.sh`? I.e. just call your shard `test-fa3` but rely on the same .yml file?